packages/react-liveness/src/components/FaceLivenessDetector/service/utils/videoRecorder.ts

```diff
@@ -64,6 +64,10 @@ export class VideoRecorder {
     this._recorder.dispatchEvent(event);
   }
 
+  getVideoChunkSize(): number {
+    return this._chunks.length;
```

Comment:
Any chance of `this._chunks` being falsy?

Reply:
shouldn't be, we initialize it to empty array